src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1985,8 +1985,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         WRN_GeneratorFailedDuringGeneration = 42502
         WRN_AnalyzerReferencesFramework = 42503
 
-        ' // AVAILABLE                             42600 - 49998
-        ERRWRN_NextAvailable = 42600
+        WRN_CallerArgumentExpressionAttributeSelfReferential = 42600
+        WRN_CallerArgumentExpressionAttributeHasInvalidParameterName = 42601
+
+        ' // AVAILABLE                             42602 - 49998
```

Comment:
>' // AVAILABLE                             42602 - 49998 [](http://example.com/codeflow?start=8&length=56)

It looks like the numbers were intentionally moved far ahead and we were not changing them recently when adding new warnings, simply were oncrementing last used value. I suggest doing the same. I will open an issue to follow up on this comment.